packages/scene-composer/src/utils/colorUtils.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * This method uses a regular expression (`hexRegex`) to validate a hex color code.
+ * The regex checks if the hex code starts with a "#" symbol, followed by either a
+ * 6-digit combination of characters from A-F, a-f, and 0-9.
+ * The `test` method is then used to validate the `hexCode` against the regex pattern.
+ * @param hexCode
+ * @returns
+ */
+export const isValidHexCode = (hexCode: string): boolean => {
```

Comment:
In general we should let ThreeJS handle colors, it provides a more flexible interface and allows for color values to be more than just hex codes (less rigid). See previous comment.